views/dashboard.handlebars

```diff
@@ -0,0 +1,62 @@
+<main class="no-padding no-pad-top taskmain">
```

Comment:
I have been linking everything to home.handlebars as the dashboard. What is dashboard.handlebars for?

Reply:
Can you include a screenshot of how this dashboard looks in the browser?